### PR TITLE
Rhel7 branch rsyslog via journald

### PR DESCRIPTION
**Pull request by rvykydal**

### CODE REVIEW

**Comment by bcl** [comment]
As long as this doesn't interfere with virtio logging or the --syslog option it looks fine to me.

File: share/config_files/common/rsyslog.conf [hunk 1]
```diff
@@ -15,35 +32,59 @@ $InputFileStateFile anaconda-tb-statefile
...
+
+# Where to place auxiliary files
+#$WorkDirectory /var/lib/rsyslog
+
 # Use default timestamp format
 $ActionFileDefaultTemplate RSYSLOG_TraditionalFileFormat
 
+# File syncing capability is disabled by default. This feature is usually not required,
+# not useful and an extreme performance hit
+#$ActionFileEnableSync on
```
Comment (vpodzime) [inline share/config_files/common/rsyslog.conf]: Do we really want this?

**Comment by vpodzime** [comment]
Looks good to me too otherwise.

File: share/config_files/common/rsyslog.conf (hunk 1 above)
Comment (rvykydal) [inline share/config_files/common/rsyslog.conf]: There is no change regarding this option.
I'd leave it for follow-up consideration.

**Comment by rvykydal** [comment]
Both remote logging and virtio channel logging features keep working with the patch.

**Pull request merged**